apps/web/src/components/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+import { useRef } from 'react';
+import './SearchBar.scss';
+import SearchIcon from '../../assets/searchIcon.svg';
+
+export interface Props {
+  /** What happens when we use a searchstring to search for results? */
+  handleSearch: (searchString: string) => void;
+  /** What happens when a user begins typing what they're searching for? */
+  filterResults: (searchString: string) => object[];
+  /** An optional custom test ID for the component.  */
+  customTestId?: string;
+}
+
+/**
+ * A component that renders a SearchBar
+ *
+ * @param {Object} props - The component props
```

Comment:
Just a note, but you can type the JSDocs more tightly. 
```suggestion
 * @param {Props} props - The component props
```